.github/workflows/auto-assign-owners.yml

```diff
@@ -3,8 +3,9 @@ on:
   pull_request_target:
     types: [opened, ready_for_review]
 
+# Do not cancel this workflow on main.
 concurrency:
-  group: ${{ github.workflow }}-${{ github.head_ref }}
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
```

Comment:
I don't think we need this change as this job doesn't run on main

Reply:
Good catch